### PR TITLE
src: warning if a file doesn't exist

### PR DESCRIPTION
**Pull request by helio-frota**
Connects to #54

### CODE REVIEW

**Comment by helio-frota** [comment]
screenshot with the warning message:
foo

**Comment by lholmquist** [comment]
👀

File: lib/docker-archiver.js [hunk 1]
```diff
@@ -30,8 +31,10 @@ async function createArchive (config) {
   const includedFiles = [];
   // Look to see if any files are specified in the package.json
   if (config.projectPackage.files && config.projectPackage.files.length > 0) {
-    // If so, use that list
-    includedFiles.push.apply(includedFiles, config.projectPackage.files);
+    // Removes all nonexistent files and use that list
+    const normalizedFileList = helpers.normalizeFileList(config.projectPackage.files);
+    logger.warning(`The following files do not exist: ${normalizedFileList.nonexistent}`);
```
Comment (lholmquist) [inline lib/docker-archiver.js]: if the `normalizedFileList.nonexistent` is zero, then maybe don't print anything

Comment (helio-frota) [inline lib/docker-archiver.js]: true! thanks!

**Pull request merged**